org.freedesktop.fwupd.json

```diff
@@ -0,0 +1,211 @@
+{
+    "app-id": "org.freedesktop.fwupd",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.28",
+    "branch": "master",
+    "sdk": "org.gnome.Sdk",
+    "command": "/app/libexec/fwupd/fwupdtool",
+    "tags": [],
+    "finish-args": [
+        "--device=all",
+        "--filesystem=/boot",
+        "--filesystem=/sys",
+        "--filesystem=xdg-download",
+        "--share=network",
+        "--system-talk-name=org.freedesktop.fwupd",
+        "--system-talk-name=org.freedesktop.UPower"
+    ],
+    "build-options" : {
+        "cflags": "-O2 -g",
+        "cxxflags": "-O2 -g"
+    },
+    "cleanup": [
+        "*.a",
+        "*.la",
+        "/include",
+        "/lib/girepository-1.0",
+        "/lib/pkgconfig",
+        "/share/bash-completion",
+        "/share/dbus-1/system-services",
+        "/share/gir-1.0",
+        "/share/gtk-doc",
+        "/share/info",
+        "/share/man",
+        "/share/pkgconfig" ],
+    "modules": [
+        {
+          /* not using shared-modules as we need gudev */
```

Comment:
Maybe we could just enable it there? But I'm fine with having it separately here.

Reply:
The shared modules tend to clean up themeslves.

Reply:
Ahh, agreed. I've opened a PR here: https://github.com/flathub/shared-modules/pull/31 -- if that is merged I'll fix up the manifest here.

Reply:
I'm using the shared-modules now, thanks for the quick review of the gudev bit.